### ISSUE TITLE
Enable to choose difficulty level

### ISSUE THREAD

**Issue by yelynn1** [yelynn1/tictactoe#12]

**Comment by Yashs911** [comment]
@yelynn1 I would like to work on this issue can you please guide me on what all changes do I need to make and in which file

**Comment by ankitgauri26** [comment]
Can i work on this?

**Comment by claricetorres** [comment]
Hey, I'd like to work on this.

**Comment by Paras-code-007** [comment]
can i work on this

**Comment by yelynn1** [comment]
We welcome any kind of contributions. Feel free to make a PR. Thanks

**Comment by Edu93Jer** [comment]
Hi @yelynn1 I would like to contribute to this issue, seems kind of ez or what you think? If no, could you please explain to me a little more about the project, thanks...

**Comment by Edu93Jer** [comment]
Hi!, @yelynn1 I saw there is a PR open from @GouravSingal-code, looks good to me, so that mean that this issue is close? or could you please tell me what else is needed to improve?

**Comment by Riddhish97** [comment]
Hey @yelynn1 I would like to contribute. This issue is still open?

**Comment by alexandrakitsa** [comment]
Hello,  i would like to contribute to this issue. Is it still open?

**Comment by yelynn1** [comment]
yes for sure. it's still opened. Feel free to make a pull request

**Comment by Trevy05** [comment]
is this still open?

**Comment by disha100** [comment]
is this still open ?
if yes can i work on this

**Comment by vinayak9303** [comment]
Hi Yelynn1, is this issue still open can I work on it?

**Comment by mayankagnihotri7** [comment]
Hey @yelynn1 Can I take this up?

**Comment by Deeshon** [comment]
hey @yelynn1 is the issue still open? Can i work on this? I cud add a simple ai to let the player play against the computer.

**Comment by Vaishali054** [comment]
Hey @yelynn1 is this issue still open, I would love to work on this

**Comment by keshavkumar03** [comment]
Hi @yelynn1  Is the issue still open to be resolved, can I work on it 
?

**Comment by screen079** [comment]
Since there is difficulty selection has this been finished?
